TheThingsNetwork/lorawan-stack
# Fix collaborator prefix for CLI create commands

#### Summary


Closes #5442 
Unblocks https://github.com/TheThingsIndustries/lorawan-stack/pull/3097

#### Changes


- Added `collaborator` prefix for `application`, `gateway` and `client` create commands.
- Upgrade `protoc-gen-go-flags` image.


#### Testing



Manual testing, pretty much running the command:
`ttn-lw-cli app create < app.json --user-id admin`
This same test for `gateway` and `client`.

Besides this also printed locally the modified flags to validate if the prefix was present:
```bash
go run ./cmd/ttn-lw-cli client create < client.json --user-id admin
INFO    Client secret: NYKXTN4ILN7ZQH7ILEDN6WBWMJP6RRHMWK74PSH4654CTJ42WMBQ
WARN    The Client secret will never be shown again
WARN    Make sure to copy it to a safe place
{
  "ids": {
    "client_id": "mock-client"
  },
  "created_at": "2022-05-13T13:21:48.379Z",
  "updated_at": "2022-05-13T13:21:48.379Z",
  "administrative_contact": {
    "user_ids": {
      "user_id": "admin"
    }
  },
  "technical_contact": {
    "user_ids": {
      "user_id": "admin"
    }
  },
  "secret": "NYKXTN4ILN7ZQH7ILEDN6WBWMJP6RRHMWK74PSH4654CTJ42WMBQ",
  "state": "STATE_APPROVED",
  "grants": [
    "GRANT_AUTHORIZATION_CODE"
  ],
  "rights": [
    "RIGHT_ALL"
  ]
}
# printed flag values
INFO    collaborator.ids.user-ids.user-id
INFO    user-id
```

##### Regressions



None that I'm aware

#### Notes for Reviewers


...

#### Checklist
